translation-web/src/main/java/eu/europeana/api/translation/web/service/RedisCacheService.java

```diff
@@ -0,0 +1,58 @@
+package eu.europeana.api.translation.web.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RedisCacheService {
+
+  private RedisTemplate<Integer, String> redisTemplate;
+
+  @Autowired
+  public RedisCacheService(RedisTemplate<Integer, String> redisTemplate) {
```

Comment:
Redis keys are strings, therefore we would need String as type for the Key, In the spike ticket we decided to include the original string into the cache, therefore we would need a CachedTraslation object to be used as type for the cached values. So RedisTemplate<String, CachedTraslation >